### DISCUSSION TITLE
Welcome https://github.com/zhaque44 to join helmfile

### DISCUSSION THREAD

**Discussion by yxxhero** [helmfile/helmfile#1678] [Ideas]
Todos:
- merged MRs links.
- self introduction.

**Comment by zhaque44** [comment]
https://github.com/helmfile/helmfile/commits?author=zhaque44

**self introduction**

Zubair Haque is a seasoned and knowledgeable SRE/Open Source Contributor professional with over 13 years of experience in cloud computing and engineering. He is proficient in designing, implementing, and maintaining cloud-based solutions in many environments. In his free time he likes to spend time with his children & enjoys exploring new foods.

**Comment by mumoshu** [comment]
Hey @zhaque44! Glad to have you on board 👋 

How did you learn about helmfile and kindly spend time contributing to it?!

For me, I started contributing helmfile back when it was [roboll/helmfile](https://github.com/roboll/helmfile) while I was working for previous companies, envisioning I'd use it for one of the building blocks for my Kubernetes-based internal platforms.

**Comment by zhaque44** [comment]
I was introduced to helmfile by my colleagues, quickly realized it's an efficient tool to deploy helm releases to diff environments with ease. I was able to get up to speed fairly quickly & was in favor of the tool being used on projects everywhere.

**Comment by xiaomudk** [comment]
Welcome aboard,  @zhaque44!  Looking forward to your contributions!
